modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/resources/META-INF/resources/item/selector/FDSAdminItemSelector.tsx

```diff
@@ -30,16 +30,34 @@ const views = [
 	},
 ];
 
-const FDSAdminItemSelector = ({
+const FDSAdminItemSelector = (
+	{
 	className,
 	classNameId,
+	classPK,
 	namespace,
 }: {
-	className: String;
-	classNameId: String;
-	namespace: String;
-}) => {
-	const [selectedItem, setSelectedItem] = useState<ISelectedItem>();
+	className: string;
+	classNameId: string;
+	classPK: string;
+	namespace: string;
+}
+) => {
+	const selectedId = window.frameElement && window.frameElement?.getAttribute('classPK');
+
+	const preselectedItem = {
```

Comment:
Build the `preselectedItem` from the props attached to the `<iframe>` in the [Modal](https://github.com/liferay-frontend/liferay-portal/pull/4362/files#diff-15984289d46b19a7d68a56b2d41c72a99fa2307cb9628c43f2d0f32f2b47129fR262)